astropy/table/ndarray_mixin.py

```diff
@@ -0,0 +1,66 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+import numpy as np
+
+from astropy.utils.data_info import ParentDtypeInfo
+
+
+class NdarrayMixinInfo(ParentDtypeInfo):
+    _represent_as_dict_primary_data = 'data'
+
+    def _represent_as_dict(self):
+        """Represent Column as a dict that can be serialized."""
+        col = self._parent
+        out = {'data': col.view(np.ndarray)}
+        return out
+
+    def _construct_from_dict(self, map):
+        """Construct Column from ``map``."""
+        data = map.pop('data')
+        out = self._parent_cls(data, **map)
+        return out
+
+
+class NdarrayMixin(np.ndarray):
+    """
+    Mixin column class to allow storage of arbitrary numpy
+    ndarrays within a Table.  This is a subclass of numpy.ndarray
+    and has the same initialization options as ndarray().
+    """
+    __module__ = 'astropy.table.table'
```

Comment:
Is it necessary to set the module? And why not to the correct path?

Reply:
I forget the details, but I think this was necessary to make this look like it is part of the main `astropy/table/table.py` module for the narrative / API docs. (In other words something went wrong that I though needed to be fixed when I didn't have this line). There is always this question of how to factor things out of a single giant module.